core/io/multiplayer_api.h

```diff
@@ -41,9 +41,8 @@ class MultiplayerAPI : public RefCounted {
 public:
 	enum RPCMode {
 		RPC_MODE_DISABLED, // No rpc for this method, calls to this will be blocked (default)
-		RPC_MODE_REMOTE, // Using rpc() on it will call method in all remote peers
-		RPC_MODE_MASTER, // Using rpc() on it will call method on wherever the master is, be it local or remote
-		RPC_MODE_PUPPET, // Using rpc() on it will call method for all puppets
+		RPC_MODE_REMOTE, // Any peer can call this rpc()
```

Comment:
Should this be renamed `RPC_MODE_ANY` to match the annotation?
It also seems tautological for a "Remote Procedure Call Mode" to be "Remote" :)

Reply:
That being said it might be ambiguous and understood as "any mode", i.e. "both Disabled and Master".

Reply:
No, RPC_MODE_ANY would be fine I think.

Reply:
done